lib/screens/dashboard.dart

```diff
@@ -34,6 +36,10 @@ class _DashboardState extends State<Dashboard> with AutomaticKeepAliveClientMixi
     setState(() {});
   }
 
+  void refreshState() {
+    setState(() {});
+  }
+
```

Comment:
Calling refreshState makes the screen go to top. Make sure you don't use it whenever someone scrolls down.